pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -390,48 +390,48 @@ func TestZeroHintMath(t *testing.T) {
 				errCheck: errorIsNil,
 			},
 		},
-		"SplitIntHint": {
+		"SplitInt": {
 			// Performs value%base and stores that to output.
 			// Asserts output<bound.
 			{
 				operanders: []*hintOperander{
-					{Name: "output", Kind: uninitialized},
+					{Name: "output", Kind: fpRelative, Value: addr(8)},
 					{Name: "value", Kind: fpRelative, Value: feltInt64(15)},
 					{Name: "base", Kind: fpRelative, Value: feltInt64(2)},
 					{Name: "bound", Kind: fpRelative, Value: feltInt64(5)},
 				},
 				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
 					return newSplitIntHint(ctx.operanders["output"], ctx.operanders["value"], ctx.operanders["base"], ctx.operanders["bound"])
 				},
-				check: varValueEquals("output", feltInt64(1)),
+				check: consecutiveVarAddrResolvedValueEquals("output", []*fp.Element{feltInt64(1)}),
```

Comment:
why is this change needed?

Reply:
ah we need a var addr resolved value not a var value

Reply:
oh yeah indeed, thanks